packages/gatsby-remark-images/README.md

```diff
@@ -52,7 +52,7 @@ plugins: [
 | `sizeByPixelDensity`   | `false` | Analyze images' pixel density to make decisions about target image size. This is what GitHub is doing when embedding images in tickets. This is a useful setting for documentation pages with a lot of screenshots. It can have unintended side effects on high pixel density artworks.<br /><br />Example: A screenshot made on a retina screen with a resolution of 144 (e.g. Macbook) and a width of 100px, will be rendered at 50px.                                                                                                                                                                                           |
 | `wrapperStyle`         |         | Add custom styles to the div wrapping the responsive images. Use the syntax for the style attribute e.g. `margin-bottom:10px; background: red;` or a function returning a style string which receives the information about the image you can use to dynamically set styles based on the aspectRatio for example.                                                                                                                                                                                                                                                                                                                  |
-| `backgroundColor`      | `white` | Set the background color of the image to match the background image of your design.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `backgroundColor`      | `white` | Set the box-shadow color of the image to match the background image of your design.<br /><br />**Note:** To completely disable box-shadow, use `backgroundColor: none`. You can also use `backgroundColor: transparent` for a transparent box-shadow (i.e., `box-shadow: inset 0px 0px 0px 400px transparent`).                                                                                                                                                                                                                                                                                                                    |
```

Comment:
@kimbaudi I think this is useful info and we should include it 👍 However, I agree with @pieh that people don't really need to know implementation details. I'd propose something like this:

```suggestion
| `backgroundColor`      | `white` | Set the background color of the image to match your design.<br /><br />**Note:** To completely remove the background color, set this option to `none`.                                                                                                                                                                                                                                                                                                                    |
```

This keeps your update on how to disable the background color, but removes the info about `backgroundColor: transparent` as that's effectively the same as `backgroundColor: none`. It also removes any mention of box-shadow, as that knowledge isn't required to understand this option. 

What do you think?

Reply:
Hi I made an edit based on you and pieh's suggestion. Thank you both! I will push the changes.